packages/kit/src/runtime/client/client.js

```diff
@@ -259,8 +259,8 @@ export async function start(_app, _target, hydrate) {
 	// connectivity errors after initialisation don't nuke the app
 	default_layout_loader = _app.nodes[0];
 	default_error_loader = _app.nodes[1];
-	default_layout_loader();
-	default_error_loader();
+	await default_layout_loader();
+	await default_error_loader();
```

Comment:
Another example of where this rule does active harm: This would slow everyone's apps down. The idea here is to start loading the layout and error eagerly in case its needed later on. But it shouldn't block the function.
```suggestion
	default_layout_loader();
	default_error_loader();
```

Reply:
It's my sloppiness doing harm. I've updated it to load these in parallel. I think it's better to be explicit about it rather than leaving a question of whether an `await` was overlooked by accident